desktop/app-handlers/crash-reporting/index.js

```diff
@@ -11,10 +11,10 @@ const debug = require( 'debug' )( 'desktop:crash-reporting' );
 /**
  * Internal dependencies
  */
-const Config = require( 'lib/config' );
+const Config = require( '../../lib/config' );
 
 module.exports = function() {
-	if ( Config.crash_reporter.electron ) {
+	if ( Config.crash_reporter && Config.crash_reporter.electron ) {
```

Comment:
I recall seeing errors from this line in my recent attempts to merge desktop > calypso so this is nice to see.
One concern that I have though: is there a chance that this'll give us false positives?
My guess is that it won't and that this change is of benefit, but I just wanted to raise the question anyway :)